src/main.cpp

```diff
@@ -1,9 +1,8 @@
-#include <iostream>
+#include "Dollar.h"
+ 
+int main(int argc, char** argv) {
 
-using namespace std;
-
-int main()
-{
-	cout << "Hello World!" << endl;
-	return 0;	
+	Dollar* dollar = new Dollar();	
```

Comment:
This is important: you're creating a pointer for the object `dollar` in the free memory, and by doing so, C++ will *not* automatically free this amount of memory. In the end of method, before the return point, you should delete the pointer by doing `delete dollar;` and `dollar = NULL;` right after. By doing this, you gonna tell the operating system that you don't need this amount of memory anymore and everyone can use it from know. Valgrind is a software that you can use in order to monitor the (de)allocation of pointers and improve the quality of your program **http://valgrind.org/**